scripts/start.sh

```diff
@@ -2,18 +2,10 @@
 
 DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" >/dev/null && pwd )"
 source ${DIR}/helper/functions.sh
-source ${DIR}/../env_files/config.env
+source ${DIR}/env.sh
```

Comment:
Now that the other params are removed, perhaps this `CLEAN` logic should be moved to sit immediately prior to line 31:

```
# <<Move lines here>>
if [[ "$CLEAN" == "true" ]] ; then
```

Reply:
Good idea, done.